pkg/arvo/lib/archive.hoon

```diff
@@ -0,0 +1,49 @@
+/-  *archive
+|%
+::
+++  export-app
+  |=  [app=@tas our=@p now=@da]
+  .^(* %gx /(scot %p our)/[app]/(scot %da now)/export/noun)
+::
+++  export-all
+  |=  [our=@p now=@da]
+  ^-  archive-0
+  :*  %0
```

Comment:
Love that we can type these exports as a whole. This is starting to get really good